app/controllers/api/posts_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::PostsController < ApplicationController
+  load_and_authorize_resource
+  def index
+    @posts = Post.all
+    render json: { success: true, message: 'Loaded all posts', data: { posts: @posts } }
+  end
+end
```

Comment:
- [ ] Currently, I'm not able to see any post, not in the seed file or the method to create. Please provide some raw data in the seed file so that I can check whether all the other API call is working.

Reply:
I added the new data that you requested. Thank you for letting me know.